plinth/modules/transmission/templates/transmission.html

```diff
@@ -0,0 +1,55 @@
+{% extends "base.html" %}
+{% comment %}
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+{% endcomment %}
+
+{% load bootstrap %}
+
+{% block content %}
+
+<h2>Bittorrent (Transmission)</h2>
+
+<p>BitTorrent is a peer-to-peer file sharing protocol.  Transmission
+   daemon handles Bitorrent file sharing.  Note that BitTorrent is not
+   anonymous.</p>
+
+<p>Access the web interface at
+   <a href="http://{{ status.hostname }}/transmission">
```

Comment:
I think a relative link should work more reliably here. (I can't generally access my freedombox using the hostname.)